app/data/generators/personal-details.js

```diff
@@ -84,7 +179,8 @@ module.exports = () => {
     y65_over: 0
   })
 
-  const dateOfBirth = dateOfBirthOptions[selectedDateOfBirth]
+  let dateOfBirth = dateOfBirthOptions[selectedDateOfBirth]
+  dateOfBirth = DateTime.fromJSDate(dateOfBirth).toFormat('yyyy-LL-dd')
```

Comment:
I'm worried about "fromJSDATE" being used.

Reply:
It's a valid date. Faker creates a time, which is unnecessary, so I've removed.